datamodel/src/test/kotlin/com/hartwig/actin/datamodel/molecular/TestMolecularFactory.kt

```diff
@@ -324,6 +366,22 @@ object TestMolecularFactory {
         isAssociatedWithDrugResistance = true,
     )
 
+    fun createMinimalFusion() = Fusion(
```

Comment:
Same comment on ordering here. Also, `fusedExonUp` and `fusedExonDown` could be null in a minimal version